deploy/olm-catalog/ibm-catalog-ui-operator/3.6.2/ibm-catalog-ui-operator.v3.6.2.clusterserviceversion.yaml

```diff
@@ -8,9 +8,6 @@ metadata:
           "apiVersion": "operator.ibm.com/v1alpha1",
           "kind": "CatalogUI",
           "metadata": {
-            "annotations": {
```

Comment:
v3.6.2 already released. Need to bump version and then make changes

Reply:
No, v3.6.1 was the last release on stable-v1
https://github.com/IBM/ibm-catalog-ui-operator/blob/master/deploy/olm-catalog/ibm-catalog-ui-operator/ibm-catalog-ui-operator.package.yaml#L6
https://github.ibm.com/IBMPrivateCloud/ibm-catalog-ui-operator-case/blob/master/stable/ibm-catalog-ui-operator-bundle/case/ibm-catalog-ui-operator/inventory/ibmCatalogUiOperatorSetup/resources.yaml#L43